app/src/main/java/kr/co/connect/boostcamp/livewhere/di/databaseModule.kt

```diff
@@ -0,0 +1,24 @@
+package kr.co.connect.boostcamp.livewhere.di
+
+import androidx.room.Room
+import kr.co.connect.boostcamp.livewhere.data.database.AppDataBase
+import org.koin.android.ext.koin.androidContext
+import org.koin.dsl.module.module
+
+const val DATABASE_NAME = "mdatabase.db"
+
+val databaseModule = module {
+    single("databaseModule") {
```

Comment:
아래와 같이 이름이 바뀌어야 하지 않을까 싶습니다.
```suggestion
    single("appDataBase") {
```